config/sidekiq.yml

```diff
@@ -10,18 +10,19 @@
 
 :schedule:
   CalculateAllMetrics:
-    cron: '0 0 * * * *' # Run at midnight
+    cron: '0 0 0 * * *' # Run at midnight
     class: CalculateAllMetricsJob
     queue: scheduled
   PreloadOpenData:
-    cron: '0 1 * * * *' # Run at 01:00
+    cron: '0 0 1 * * *' # Run at 01:00
     class: PreloadOpenDataJob
     queue: scheduled
   OrdersReminder:
-    cron: '0 18 * * * *' # Run at 18:00
+    cron: '0 0 18 * * *' # Run at 18:00
```

Comment:
Here comment are not representative of the current cron config 
'0 0 18 * * *' => "At 12:00 AM, on day 18 of the month"
https://crontab.cronhub.io/

Reply:
Your generator doesn't give me the same answer ...
![image](https://user-images.githubusercontent.com/464350/145186102-cd2967bc-50a9-442c-9ed2-6a00ec91841a.png)